### PR TITLE
Fix contexture repo links

### PR DESCRIPTION
**Pull request by stellarhoof**
Point to new packages and while we're at it also point to `main` (all hail the new master!)

### CODE REVIEW

File: packages/react/README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Documentation lives in our [Storybook](https://smartprocure.github.io/contexture
 
 - [Theming documentation](https://smartprocure.github.io/contexture-react/?path=/story/developer-notes-docs--theme-api)
 - A [Live Demo](https://smartprocure.github.io/contexture-react/?path=/story/live-demos-imdb-search--dynamic-search-layout) using IMDB data
```
Comment (daedalus28) [inline packages/react/README.md:52]: These are interesting - how are we going to handle per package sites? I guess we can try to put it all in one place in one large site, but storybook is really kind of its own thing 🤔

**Review by daedalus28 — CHANGES_REQUESTED** [review]
I think we also need to bump all the packages so npm updates

File: packages/react/README.md (hunk 1 above)
Comment (stellarhoof) [reply packages/react/README.md:52]: We can deploy under a path in the site for this repo. Also related: https://github.com/github/pages-gem/issues/724

Comment (stellarhoof) [reply packages/react/README.md:52]: https://github.com/github/pages-gem/issues/724

**Comment by stellarhoof** [comment]
I'll add changesets first and test-drive it by bumping patch versions here

**Review by daedalus28 — APPROVED** [review]

**Pull request merged**